crates/rspack_core/Cargo.toml

```diff
@@ -8,6 +8,7 @@ version     = "0.2.0"
 [dependencies]
 anymap = { workspace = true }
 async-recursion = { workspace = true }
+async-scoped = { workspace = true }
```

Comment:
Supposed to be removed?

Reply:
not yet, still a case use this and i need to discuss further about @quininer how to migrate all  exsiting code to future_scope

Reply:
i need to remove this to support wasm

Reply:
removed